.circleci/config.yml

```diff
@@ -2,10 +2,7 @@ version: 2.1
 executors:
   node14:
     docker:
-    - image: circleci/node:14
-    environment:
-      NPM_CONFIG_PREFIX: ~/.npm-global
-      LC_ALL: en_US.UTF-8
+    - image: cimg/node:16.15
```

Comment:
replaced with node16, NPM_CONFIG_PREFIX no longer needed (and causes an error on startup)